### PR TITLE
Optionalstuff

### PR DESCRIPTION
**Pull request by voisardf**
Please review but DO NOT merge before PR https://github.com/sitn/crdppf_core/pull/68 is merged. Then I'll merge #68 into this and resolve conflicts before continuing.

### CODE REVIEW

**Comment by voisardf** [comment]
conflicts resolved. @kalbermattenm : please review when available =:0)

File: crdppf/templates/derived/init.js [hunk 1]
```diff
@@ -78,12 +78,19 @@ Crdppf.baseLayersList = {'baseLayers': [
...
-    Crdppf.defaultTiles = {
-        'wmtsname' : "${baseLayers[0]['wmtsname']}",
-        'tile_format' : "${baseLayers[0]['tile_format']}"
-    }
+
+% if len(baseLayers) > 0:
+    Crdppf.defaultTiles = Crdppf.baseLayersList['baseLayers'][0];
+% else:
+    % if 'defaultTiles' in request.registry.settings.keys() :
+        Crdppf.defaultTiles = {${request.registry.settings['defaultTiles']|n}};
```
Comment (kalbermattenm) [inline crdppf/templates/derived/init.js]: You have to indent javascript and mako independently, thus, here, you javascript code is too indented

File: crdppf/util/get_feature_functions.py [hunk 2]
```diff
@@ -40,18 +40,17 @@ def get_features_function(parcelGeom, params):
                 if geometryType == 'ST_Polygon' or geometryType == 'ST_MultiPolygon':
                     intersectionMeasure = DBSession.scalar(feature.geom.intersection(parcelGeom).area())
                     if intersectionMeasure >= 1:
-                        intersectionMeasureTxt = str(math.ceil(intersectionMeasure*10)/10) + ' [m2]'
+                        intersectionMeasureTxt = ' : '+ str(int(intersectionMeasure)) + ' [m2]'
```
Comment (kalbermattenm) [inline crdppf/util/get_feature_functions.py]: Is that string going to javascript/html ? If yes, you could eventually improve it by: `[m<sup>2</sup>]`

Just a suggestion...

**Comment by kalbermattenm** [comment]
Just some suggestion for some small improvements, I'll let you choose to apply them or not.

Please merge when happy ;-)

File: crdppf/util/get_feature_functions.py (hunk 2 above)
Comment (voisardf) [inline crdppf/util/get_feature_functions.py]: unfortunately not only. But I'll give it a try anyway in case fpdf accepts it also. :)

**Comment by voisardf** [comment]
I'm quite a happy person, so merging ;0)
thanks a lot @kalbermattenm

**Pull request merged**